src/js/models/search/SearchOperation.js

```diff
@@ -115,6 +116,22 @@ class SearchOperation {
                 this.locationDomesticForeign, this.searchContext));
         }
 
+        // Add Budget Category queries
```

Comment:
If we add these to `commonParams`, let's take them out of the `uniqueParams` in the subclassed Search Operations.